cypress-shared/support/affirm-payment/outputvalidation.js

```diff
@@ -0,0 +1,56 @@
+import { PRODUCTS } from '../common/utils'
+
+export default {
+  orderInfo: {
+    inboundUrl:
```

Comment:
Do we need this orderInfo property

Reply:
@Syed-Vtex  saravanan has using this property in graphql testcases

Reply:
We are moving OrderUpdate graphql testcase to intercept
So, it is not required 
I have removed it